test/dlc_tests/ec2/pytorch/training/test_pytorch_training.py

```diff
@@ -171,6 +172,24 @@ def test_pytorch_amp(pytorch_training, ec2_connection, gpu_only, ec2_instance_ty
     execute_ec2_training_test(ec2_connection, pytorch_training, PT_AMP_CMD)
 
 
+@pytest.mark.integration("pt_s3")
+@pytest.mark.model("N/A")
+@pytest.mark.parametrize("ec2_instance_type", PT_EC2_GPU_INSTANCE_TYPE, indirect=True)
+def test_pytorch_s3(pytorch_training, ec2_connection, gpu_only, ec2_instance_type):
```

Comment:
Please name this `test_pytorch_s3_gpu`, and the second test `test_pytorch_s3_cpu`.

Reply:
Done

Reply:
This doesn't seem to have been updated. Perhaps there is a commit that hasn't been pushed?

Reply:
Updated again